examples/dev/trace-analytics-sample-app/docker-compose.yml

```diff
@@ -59,6 +59,7 @@ services:
     volumes:
       - ./resources/data-prepper-wait-for-odfe-and-start.sh:/usr/share/data-prepper/data-prepper-wait-for-odfe-and-start.sh
       - ./resources/trace_analytics_no_ssl.yml:/usr/share/data-prepper/data-prepper.yml
+      - ./resources/data_prepper_config.yml:/usr/share/data-prepper/data_prepper_config.yml
```

Comment:
Would recommend calling the pipeline config "pipelines.yml" and the data prepper config "data-prepper.yml" if it's not too much trouble

Reply:
Np. This is a comeby and fix.

Reply:
After a second thought shall we push it back later. b/c right now it is consistent with the release and example.

Reply:
Ah good point, after 0.8.0 is released/merged then we can go back and update